src/sentry/models/integration.py

```diff
@@ -13,19 +13,35 @@
 from sentry.signals import integration_added
 
 
-class PagerDutyServiceProject(Model):
+class PagerDutyService(Model):
     __core__ = False
 
-    project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False)
     organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration")
     integration_key = models.CharField(max_length=255)
     service_id = models.CharField(max_length=255)
     service_name = models.CharField(max_length=255)
```

Comment:
Could you add our standard `date_added` field to this and `PagerDutyServiceProject`